metabase/metabase
# fix miscalculation of y ticks width

`XYChart` has a miscalculation of y ticks width because it invoked a JS function with incorrect parameters. 

### How to verify
- Open http://localhost:3000/_internal/static-viz
- Ensure the Y-axis of the `Line/Area/Bar chart with negative values, different X ranges, and right Y-axis` chart does not overlap with the X-axis.

Before
<img width="691" alt="Screen Shot 2021-12-03 at 22 11 23" src="https://user-images.githubusercontent.com/14301985/144652364-e1706eaf-0549-4bcb-80d1-0b02da41fe80.png">

After
<img width="659" alt="Screen Shot 2021-12-03 at 22 10 42" src="https://user-images.githubusercontent.com/14301985/144652386-e3ae303f-a36c-4934-b95f-de77aa66c96f.png">
